templates/liquibase-pro-main.template.yaml

```diff
@@ -316,8 +319,19 @@ Parameters:
       - 'yes'
       - 'no'
     Default: 'no'
+  CodePipelineS3BucketName:
+    Description: >-
+      Name of the S3 bucket to store artifacts from CodePipeline.
+      If you leave this parameter empty, a new bucket will be created for you via CloudFormation.
+      If using a CloudFormation-generated bucket, the contents of the bucket must be deleted prior to deleting the stack.
+      This name can include numbers, lowercase letters,
+      uppercase letters, and hyphens, but do not start or end with a hyphen (-).
+      See https://aws-quickstart.github.io/option1.html.
+    Type: String
+    Default: ""
 Conditions:
   UsingDefaultBucket: !Equals [!Ref QSS3BucketName, aws-quickstart]
+  IsCodePipelineS3BuckeEmpty:  !Equals [!Ref "CodePipelineS3BucketName", ""]
```

Comment:
Minor nit: Condition isn't used in this template, which triggers a W8001 cfn-lint warning.

```
[cfn-lint] W8001: Condition IsCodePipelineS3BuckeEmpty not used
```